distributed/protocol/tests/test_highlevelgraph.py

```diff
@@ -155,6 +155,7 @@ def fn(k):
     assert plugin.resource_matches == B.npartitions
 
 
+@pytest.mark.xfail(dd._dask_expr_enabled(), reason="Annotation is WIP in dask-expr")
 @gen_cluster(client=True)
 async def test_dataframe_annotations(c, s, a, b):
```

Comment:
see also https://github.com/dask-contrib/dask-expr/pull/947

Reply:
Okay, updated to check for that warning instead now. https://github.com/dask/distributed/pull/8560/commits/302b8bb0e9d614a79817c1c7084d197acb891e24